pkg/kubectl/cmd/util/printing.go

```diff
@@ -32,6 +32,7 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().String("output-version", "", "Output the formatted object with the given version (default api-version).")
 	cmd.Flags().Bool("no-headers", false, "When using the default output, don't print headers.")
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
+	cmd.Flags().String("sort-by", "", "If non-empty, sort list types using this field.  The field must be an integer or a string, the template format is JSONPath, (e.g. '.ObjectMeta.Name')")
```

Comment:
This actually only supports jsonpath at the moment (and not integer column numbers), doesn't it?